video/out/vo_gpu_next.c

```diff
@@ -841,7 +841,11 @@ static void apply_target_options(struct priv *p, struct pl_frame *target)
     int dither_depth = opts->dither_depth;
     if (dither_depth == 0) {
         struct ra_swapchain *sw = p->ra_ctx->swapchain;
-        dither_depth = sw->fns->color_depth ? sw->fns->color_depth(sw) : 0;
+        if (sw->fns->color_depth) {
+            dither_depth = sw->fns->color_depth(sw);
+        } else if (!pl_color_transfer_is_hdr(target->color.transfer)) {
```

Comment:
hmm, another question. Isn't `target->color.transfer` left unknown/auto by default? That would mean this override would work only if `--target-trc` is used. I think still good this way, but doesn't really fix/affect nominal case of default config.

Reply:
https://github.com/haasn/libplacebo/blob/master/src/colorspace.c#L318

Unknown is assumed to be 1.0, is this something that could change in the future?

Reply:
Ah, so it kinda works, but only because unknown is implementation defined. Well, changing that would probably be breaking and it make sens to default to "SDR". I guess we can try it. And see if someone complains.